src/sagemaker/tensorflow/README.rst

```diff
@@ -15,485 +34,130 @@ is a two-step process.
 
-To use a pre-canned ``tensorflow.estimator`` instead of creating a ``model_fn``, you need to write a ``estimator_fn``.
-The base skeleton for the ``estimator_fn`` looks like this:
+1. Make sure your script can handle ``--model_dir`` as an additional command line argument. If you did not specify a
+location when the TensorFlow estimator is constructed a S3 location under the default training job bucket will be passed
+in here. Distributed training with parameter servers requires you use the ``tf.estimator.train_and_evaluate`` API and
+a S3 location is needed as the model directory during training. Here is an example:
 
 .. code:: python
 
-  def estimator_fn(run_config, hyperparameters):
-    # Logic to the following:
-    # 1. Defines the features columns that will be the input of the estimator
-    # 2. Returns an instance of a ``tensorflow.estimator`` passing in, the input run_config in the
-    #    constructor.
+    estimator = tf.estimator.Estimator(model_fn=my_model_fn, model_dir=args.model_dir)
+    ...
+    train_spec = tf.estimator.TrainSpec(train_input_fn, max_steps=1000)
+    eval_spec = tf.estimator.EvalSpec(eval_input_fn)
+    tf.estimator.train_and_evaluate(mnist_classifier, train_spec, eval_spec)
 
-Example of a complete ``estimator_fn``
-''''''''''''''''''''''''''''''''''''''
+2. Load input data from the input channels. The input channels are defined when ``fit`` is called. For example:
 
 .. code:: python
 
-  def estimator_fn(run_config, hyperparameters):
-      # Defines the features columns that will be the input of the estimator
-      feature_columns = [tf.feature_column.numeric_column(INPUT_TENSOR_NAME, shape=[4])]
-      # Returns the instance of estimator.
-      return tf.estimator.DNNClassifier(feature_columns=feature_columns,
-                                        hidden_units=[10, 20, 10],
-                                        n_classes=3,
-                                        config=run_config)
-
-More details on how to create a ``tensorflow.estimator`` can be find in `Creating Estimators in tf.estimator <https://www.tensorflow.org/extend/estimators>`_.
-
-An example on how to create a TensorFlow training script with an ``estimator_fn`` can find in this `example <https://github.com/awslabs/amazon-sagemaker-examples/tree/master/sagemaker-python-sdk/tensorflow_iris_dnn_classifier_using_estimators>`_.
+    estimator.fit({'train':'s3://my-bucket/my-training-data',
+                  'eval':'s3://my-bucket/my-evaluation-data'})
 
-
-Using a ``Keras`` model instead of a ``model_fn``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-``tf.keras`` is an full implementation inside TensorFlow of the Keras API. To use a ``tf.keras``
-model for training instead of ``model_fn``, you need to write a ``keras_model_fn``. The base skeleton of
-a ``keras_model_fn`` looks like this:
+   In your training script the channels will be stored in environment variables ``SM_CHANNEL_TRAIN`` and
```

Comment:
make sure the indentation/spacing is right here - this paragraph is getting rendered as a code block with ll. 113-114